features/dd-sdk-android-trace-internal/src/main/java/com/datadog/trace/core/PendingTrace.java

```diff
@@ -223,6 +223,14 @@ void registerSpan(final DDSpan span) {
         }
     }
 
+    public void unregisterSpan(final DDSpan span){
```

Comment:
> [!IMPORTANT]
> Another point that I want to pay attention to: Dropping span wasn't supported in `CoreTracer` and I am not sure that removing span from `ROOT_SPAN` is the right thing here. Please let me know if I missing something

Reply:
Hmm...not sure it's ok here, I am thinking about the case where you have a parent span already and you only want to create the Okhttp span for the ids. In this case dropping the OkHttp span will completely drop all the trace ?